common/mariadb/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: mariadb
-version: 0.12.1
+version: 0.13.0
```

Comment:
The version is already used [in this PR](https://github.com/sapcc/helm-charts/pull/6877/files#diff-8d953323a432c852ed44578d9c4fc78801db45169ed17cbcd0af91128fe4eb14R4)

Reply:
Updated to 0.14.0